tests/datasets/test_mnist.nim

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2018 the Arraymancer contributors
+# Distributed under the Apache v2 License (license terms are at http://www.apache.org/licenses/LICENSE-2.0).
+
+import ../../src/arraymancer
+import unittest
+
+suite "Datasets - MNIST":
+  test "Load MNIST":
+    let mnist = load_mnist()
```

Comment:
mmmh, I have to figure out a way to cache that for CI, I don't think it's cool to download that every commits.

Reply:
I changed this to cache by default in `.cache/arraymancer/` - I also added `.cache` to appveyor